drivers/soundwire/bus.c

```diff
@@ -237,7 +237,7 @@ static inline int do_transfer_defer(struct sdw_bus *bus,
 	init_completion(&defer->complete);
 
 	for (i = 0; i <= retry; i++) {
-		resp = bus->ops->xfer_msg_defer(bus, msg, defer);
```

Comment:
@plbossart it seems like even the msg arg is redundant isnt it? You already set the defer->msg = msg just a couple of lines above

Reply:
just look at the patch, it's removed in the 4th....

Reply:
I'm sorry @plbossart. I'm not sure im following. Even after the 4th patch, "msg" is still an argument for the xfer_msg_defer op. What am I missing?

Reply:
oh sorry, I thought you talked about the defer argument....

Reply:
done in new 5th patch